labeller/labeller.go

```diff
@@ -102,10 +103,15 @@ func New(config *Config, kubeconfig string, logger log.Logger) (*Labeller, error
 	l.queue = workqueue.NewNamedRateLimitingQueue(workqueue.DefaultControllerRateLimiter(), "labeller")
 
 	multiNamespacePodListWatcher := NewMultiNamespaceListerWatcher(
-		kclient.CoreV1().RESTClient(),
-		"pods",
 		l.namespaces,
-		fields.Everything(),
+		func(namespace string) cache.ListerWatcher {
```

Comment:
We're making `ListerWatchers` directly now?

Reply:
It's so we can pass the fakes' list/latch function directly for testing. I don't think the `RESTClient` that was being passed to `NewListerWatcherFromClient` to generate a `ListerWatcher`s was being initialized correctly.